### PR TITLE
Npc sheet rework 2

### PR DESCRIPTION
**Pull request by guswadsworth12**
Changed the Header for the NPC to look closer to the rest of the sheets.

### CODE REVIEW

File: templates/actor/parts/actor-base-header.hbs [hunk 1]
```diff
@@ -1,11 +1,11 @@
-<header class="sheet-header">
+<header class="npc-sheet-header">
   <div class="npc-header-container">
     <div class="npc-image">
-    <img class="profile-img" src="{{actor.img}}" data-edit="img" title="{{actor.name}}"/>
+    <img class="profile-img" src="{{actor.img}}" data-edit="img" />
     </div>
-    <div class="header-fields">
+    {{!-- <div class="header-fields">
```
Comment (phildominguez) [inline templates/actor/parts/actor-base-header.hbs:6]: Can remove the commented out code

File: templates/actor/actor-zord-sheet-old.hbs [hunk 2]
```diff
@@ -1,5 +1,5 @@
 <form class="{{cssClass}} {{actor.type}} flexcol" autocomplete="off">
```
Comment (phildominguez) [inline templates/actor/actor-zord-sheet-old.hbs:1]: Can we remove this file?

**Review by phildominguez — APPROVED** [review]
Looks super good! Two comments to remove unused stuff but I'll leave it up to you.

File: templates/actor/parts/actor-base-header.hbs (hunk 1 above)
Comment (guswadsworth12) [reply templates/actor/parts/actor-base-header.hbs:6]: removed

File: templates/actor/actor-zord-sheet-old.hbs (hunk 2 above)
Comment (guswadsworth12) [reply templates/actor/actor-zord-sheet-old.hbs:1]: Removed

**Pull request merged**